packages/puppeteer-core/src/node/ScreenRecorder.ts

```diff
@@ -102,9 +128,10 @@ export class ScreenRecorder extends PassThrough {
         // VP9 tries to use all available threads?
         ['-threads', '1'],
         // Specifies the frame rate we are giving ffmpeg.
-        ['-framerate', `${DEFAULT_FPS}`],
+        ['-framerate', `${fps}`],
         // Specifies the encoding and format we are using.
-        this.#getFormatArgs(format ?? 'webm'),
+        // FIXME https://sentry.io/answers/node-js-no-overload-matches-this-call-error-with-typescript
```

Comment:
What does this `FIXME` refer to? Is it still needed?

Reply:
Not sure how that one slipped through my version control. There seems to be a bug in _GitKraken_ that sometimes sneakily includes extra adjacent lines…